Sources/Graphaello/Extensions/Path+templates.swift

```diff
@@ -20,9 +20,31 @@ extension Path {
 }
 
 private func templatePath(file: StaticString = #file) -> Path {
+	
+	let templatesPath = Path("templates")
```

Comment:
Ah!!! I see it now. Let me check if it works after being installed and get back to you

Reply:
Further I did some improvements for the creation and usage of the template folder. You has create a sub folder `graphaello`, but I think this folder isn't necessary.
I use Mint (https://github.com/yonaskolb/Mint) as dependency manager for Swift CLI and packages. For this manager all thinks works well.